tests/e2e_tests/test_data/compatibility/populate_db.py

```
@@ -65,14 +67,15 @@ def create_collection(name: str, memmap_threshold_kb: int, on_disk: bool, dataty
 
 
 def create_payload_indexes(name: str, on_disk_payload_index: bool):
-    # keyword
+    # keyword, with the extra on-disk structure backing `match: {"prefix": ...}`
     response = requests.put(
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)